content/applications/marketing/events/event_tracks.rst

```diff
@@ -0,0 +1,303 @@
+   drop-down menu to reveal hidden smart buttons. Then, click :guilabel:`Tracks` from the resulting
+   drop-down menu.
+
+Clicking that smart button reveals the :guilabel:`Event Tracks` page for that particular event,
+which presents all the tracks (both scheduled and proposed) for the event, if there are any.
+
+.. image:: event_tracks/event-tracks-page.png
+   :align: center
+   :alt: Typical event tracks page for an event in the Odoo Events application.
+
+The tracks are presented in a default :icon:`oi-view-kanban` :guilabel:`Kanban` view, but there is
+also the option to view these tracks in a :icon:`oi-view-list` :guilabel:`List`,
+:icon:`oi-view-cohort` :guilabel:`Gantt` chart, :icon:`fa-calendar-check-o` :guilabel:`Calendar`,
+:icon:`fa-area-chart` :guilabel:`Graph`, or :icon:`fa-clock-o` :guilabel:`Activity` view. All of
+which are accessible in the upper-right corner of the :guilabel:`Tracks` page.
+
+In the default :icon:`oi-view-kanban` :guilabel:`Kanban` view, the tracks are categorized into
+different stages. The default stages are: :guilabel:`Proposal`, :guilabel:`Confirmed`,
+:guilabel:`Announced`, :guilabel:`Published`, :guilabel:`Refused` (collapsed stage), and
+:guilabel:`Cancelled` (collapsed stage). All of which can be edited, if needed.
+
+.. tip::
+   To edit a stage, hover over the stage name, click the :icon:`fa-cog` :guilabel:`Settings` icon,
+   and select :guilabel:`Edit` from the resulting drop-down menu.
+
+Clicking into a track from the :guilabel:`Event Tracks` page reveals the track form for that
+particular track.
+
+Create event track
+------------------
+
+From the :guilabel:`Event Tracks` page, click :guilabel:`New` in the upper-left corner to reveal a
+blank event track form to create an event track.
+
+.. image:: event_tracks/event-track-form.png
+   :align: center
+   :alt: Typical event track form in the Odoo Events application.
+
+Start by giving this track a :guilabel:`Title`. This field is **required** by Odoo.
+
+Then, if desired, add an accompanying image to the track via the :icon:`fa-pencil`
+:guilabel:`(pencil)` icon that appears when the cursor hovers over the :icon:`fa-camera`
+:guilabel:`(camera)` icon in the upper-right corner of the form. When clicked, proceed to upload the
+desired image for the track. This image appears on the front-end of the event website, on this
+specific tracks webpage.
+
+Next, enter a :guilabel:`Track Date` and time for the track, and designate a :guilabel:`Location`
+where the talk is planning to be held.
+
+.. tip::
+   To access a complete list of locations for event tracks, which can be modified (and addded to) at
+   any time, navigate to :menuselection:`Events app --> Configuration --> Track Locations`.
+
+Then, add a :guilabel:`Duration` to the track (in minutes).
+
+If the :guilabel:`Live Broadcast` setting has been enabled in the *Events* app settings, the option
+to add a corresponding link in the :guilabel:`YouTube Video Link` field is available.
```

Comment:
Remove guilabel since this is not on the screen
```suggestion
If the *Live Broadcast* setting has been enabled in the *Events* app settings, the option to add a
corresponding link in the :guilabel:`YouTube Video Link` field is available.
```